ccd-definition/CaseEventToComplexTypes/CreateClaim.json

```diff
@@ -582,6 +582,17 @@
     "FieldDisplayOrder": 4,
     "DisplayContext": "OPTIONAL"
   },
+  {
+    "ID": "SolicitorCreate",
+    "CaseEventID": "CREATE_CLAIM",
+    "CaseFieldID": "applicant1OrganisationPolicy",
+    "ListElementCode": "PrepopulateToUsersOrganisation",
+    "DefaultValue": "YES",
+    "FieldDisplayOrder": 5,
+    "DisplayContext": "OPTIONAL",
+    "FieldShowCondition": "claimType=\"IMPOSSIBLE_CLAIM_TYPE\"",
```

Comment:
can we use `DO_NOT_SHOW_IN_UI` as we do for all permanently hidden field?

Reply:
happy to do this as a separate refactoring, this was actually a copy and paste from a field higher up in this file

Reply:
https://tools.hmcts.net/jira/browse/CMC-1720